### PR TITLE
Loading spinner + a few additional UI changes

### PR DESCRIPTION
**Pull request by swongg**
- add spinner for both retrieving the secret and creating the secret #71 
- update design of shareable link modal
- update browser title to 'Whisper' #69 
- add copy button for retrieve-secret page #72 

Retreiving Secret
1) Loading
image

2) Post-loading
image

Creating Secret
1) Loading
image

2) Post-loading
image

### CODE REVIEW

**Comment by tamf** [comment]
The issue is for fetching (retrieving) the secret, not for the link generation. Although it is fine for both of them to have the spinner.

**Comment by swongg** [comment]
> The issue is for fetching (retrieving) the secret, not for creating (storing) it. Although it is fine for both of them to have the spinner.

Added spinner for retrieving the secret. See updated comment (at the top) for images of UI changes.

File: public/script.js [hunk 1]
```diff
@@ -292,11 +307,18 @@ function clearDataOnClick() {
   }
 }
 
-function copyToClipBoard() {
+function copyLinkToClipBoard() {
```
Comment (swongg) [inline public/script.js:310]: - copyLinkToClipBoard and copySecretToClipBoard has duplicated code
- I thought I would be able to pass in the element id as a parameter (ex: pass in "secret-message-box")
but that does not work

**Review by tamf — APPROVED** [review]

File: public/script.js (hunk 1 above)
Comment (tamf) [reply public/script.js:310]: feel free to refactor it as part of a separate PR, but it's not crucial.

**Review by tamf — APPROVED** [review]

**Pull request merged**